isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -374,13 +374,20 @@ namespace Isis {
                                                   ltState, targetRadius);
 
     m_sunPosition = new SpicePosition(10, m_target->naifBodyCode());
-    
+
 
     // Check to see if we have nadir pointing that needs to be computed &
-    // See if we have table blobs to load 
+    // See if we have table blobs to load
     if (m_usingAle) {
-      m_sunPosition->LoadCache(isd["sun_position"]);
-      m_bodyRotation->LoadCache(isd["body_rotation"]);
+      m_sunPosition->LoadCache(isd["SunPosition"]);
```

Comment:
These names changed back to the old names. It looks like that is breaking things.